test/testlib/minikube_utilities.go

```diff
@@ -835,7 +836,28 @@ func VerifyLicenseIsCommunity(t *testing.T, namespace string, podName string) {
 
 	output, err := k8s.RunKubectlAndGetOutputE(t, options, "exec", "-c", "admin", podName, "--", "nuocmd", "show", "domain")
 	require.NoError(t, err)
-	require.True(t, strings.Contains(output, "server license: Community"), output)
+
+	RunOnNuoDBVersionCondition(t, ">=6.0.0", func(version *semver.Version) {
+		require.True(t, strings.Contains(output, "server license: Limited"), output)
+	})
+
+	RunOnNuoDBVersionCondition(t, "<6.0.0", func(version *semver.Version) {
+		require.True(t, strings.Contains(output, "server license: Community"), output)
+	})
```

Comment:
There is a behavior change that might break the test calling this function. E.g. before 6.0.0, the test will succeed without installing a license but now a license needs to be installed to get "server license: Limited".
I would suggest deprecating/deleting the `VerifyLicenseIsCommunity` function and replacing it with `VerifyLicense` which accepts the license type. For example:

```go
const (
	UNLICENSED LicenseType = "Unlicensed"
	LIMITED    LicenseType = "Limited"
	ENTERPRISE LicenseType = "Enterprise"
)

func VerifyLicense(t *testing.T, namespace string, podName string, expected LicenseType) {
	options := k8s.NewKubectlOptions("", "", namespace)

	output, err := k8s.RunKubectlAndGetOutputE(t, options, "exec", "-c", "admin", podName, "--", "nuocmd", "show", "domain")
	require.NoError(t, err)

	expectedString := fmt.Sprintf("server license: %s", expected)
	if expected != ENTERPRISE {
		RunOnNuoDBVersionCondition(t, "<6.0.0", func(version *semver.Version) {
			// Prior to NuoDB v6.0.0, Community and Enterprise license types are
			// supported
			expectedString = "server license: Community"
		})
	}
	require.True(t, strings.Contains(output, expectedString), output)
}
```